_posts/2023-07-05-lts-releases.adoc

```diff
@@ -0,0 +1,47 @@
+
+LTS releases offer an opportunity for our community to enjoy a version of Quarkus, with the added assurance of critical patches, bug fixes, and security updates. We understand that not every user wants to continuously upgrade to the latest release and some prefer to stick with a version they know works for them. Until now Red Hat Build of Quarkus provided such option, but we want to ensure Quarkus community project and especially community projects built on top of Quarkus has similar options.
+
+## Quarkus Users
+
+Here's what you need to know as a Quarkus user:
+
+. *Flexibility*: We still recommend users to upgrade early and often, but users now have the choice to upgrade to the latest feature-packed version or stick with the stability and predictability of an LTS release. This decision can be made based on individual requirements and risk tolerance.
+
+. *Stability*: LTS versions will be supported for a period of 12 months, allowing users to maintain the same version of Quarkus without worrying about new features introducing regressions or breaking changes. We aim to release a new LTS version every 6 months.
+
+. *Long-Term Support*: LTS versions will continue to receive critical bug fixes, security patches, and performance enhancements even after the introduction of newer versions. Critical issues that directly impact functionality, performance, or security will be prioritized for fixes.
+
+. *Security*: Regular security updates will be part of LTS releases, ensuring your applications stay protected against known vulnerabilities.
+
+. *Identification*: LTS releases will for now be identifed on https://quarkus.io/security. We are working on adding a new badge to the Quarkus website to make it easier to identify LTS releases.
+
+. *Implications for Non-LTS Releases*: Our regular monthly'ish release cycle will continue as before, providing feature updates and improvements at a high pace for those who prefer to stay on the cutting edge.
+
+## Quarkus Extension Maintainers and Contributors
+
+All of the above applies to Quarkus extension maintainers and contributors as well. In addition, we will be recommending that extension maintainers and contributors considers bug fixes and enhancements for LTS releases. This will ensure that LTS releases are as stable and robust as possible.
+
+This means that extension maintainers and contributors will need to consider having branches and versioning in place for LTS releases. 
+
+This is something many extensions already have done in context of Quarkus 2 to 3 move. For example neo4j have a 1.x for Quarkus 2 and 2.x branch for Quarkus 3. Going forward we will recommend that extensions have branch for the LTS version (currently 3.2.x) and one for main Quarkus 3 features, a 3.x.
```

Comment:
Wrong place to announce the 3.2 is the LTS.

Reply:
it's not in the text, only in the summary.

Shouldn't we repeat it? 

Typically, I missed it.

Reply:
added explicit call out